### PR TITLE
Improved HTTP proxies chapter

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

File: _includes/manuals/nightly/5.9_http_proxy.md [hunk 1]
```diff
@@ -1,8 +1,32 @@
...
+*Global option*
 
- * `:http_proxy:` - The hostname, port (and Authentication if required) as `http://[user:password@]HOST:PORT`
- * `:http_proxy_except_list:` - An array of hostnames
+In Administer - Settings - General menu there are two options available for
+setting outgoing HTTP(s) proxy for all communication (e.g. to Smart Proxies,
+Compute Resources):
 
- Both settings can also be accessed by administrators in the web interface under _Adminsiter -> Settings_
+* HTTP(S) proxy - hostname, port and optional credentials in format of `http://[user:password@]HOST:PORT`
```
Comment (melcorr) [inline _includes/manuals/nightly/5.9_http_proxy.md]: in the format of

File: _includes/manuals/nightly/5.9_http_proxy.md [hunk 2]
```diff
@@ -1,8 +1,32 @@
...
+* HTTP(S) proxy except hosts - array of hostnames to exclude
+
+Both settings can also be configured via `settings.yaml` file via
+`:http_proxy:` and `:http_proxy_except_list:` options.
+
+*HTTP Proxies page*
+
+It is possible to configure multiple HTTP(s) proxies for various Compute
+Resources. Each individual Compute Resrouce can be associated with a different
```
Comment (melcorr) [inline _includes/manuals/nightly/5.9_http_proxy.md]: Typo: Resource

File: _includes/manuals/nightly/5.9_http_proxy.md [hunk 3]
```diff
@@ -1,8 +1,32 @@
...
+proxy. To create proxy URL, visit Infrastructure - HTTP Proxies and then
+associate the proxy with a Compute Resource.
+
+Currently HTTP Proxies are supported by the following Compute Resources:
+
+* EC2
+
+*Notes*
+
+Both cases only affects outgoing HTTP(s) connection of the Foreman core
```
Comment (melcorr) [inline _includes/manuals/nightly/5.9_http_proxy.md]: affect

File: _includes/manuals/nightly/5.9_http_proxy.md [hunk 4]
```diff
@@ -1,8 +1,32 @@
...
+applicatin (Ruby on Rails process). There are other plugins and components
```
Comment (melcorr) [inline _includes/manuals/nightly/5.9_http_proxy.md]: application

File: _includes/manuals/nightly/5.9_http_proxy.md [hunk 5]
```diff
@@ -1,8 +1,32 @@
...
 
- Both settings can also be accessed by administrators in the web interface under _Adminsiter -> Settings_
+* HTTP(S) proxy - hostname, port and optional credentials in format of `http://[user:password@]HOST:PORT`
+* HTTP(S) proxy except hosts - array of hostnames to exclude
+
+Both settings can also be configured via `settings.yaml` file via
+`:http_proxy:` and `:http_proxy_except_list:` options.
+
+*HTTP Proxies page*
+
```
Comment (melcorr) [inline _includes/manuals/nightly/5.9_http_proxy.md:17]: Perhaps here you could tell the user where this is?

File: _includes/manuals/nightly/5.9_http_proxy.md [hunk 6]
```diff
@@ -1,8 +1,32 @@
...
 
- * `:http_proxy:` - The hostname, port (and Authentication if required) as `http://[user:password@]HOST:PORT`
- * `:http_proxy_except_list:` - An array of hostnames
+In Administer - Settings - General menu there are two options available for
+setting outgoing HTTP(s) proxy for all communication (e.g. to Smart Proxies,
+Compute Resources):
 
- Both settings can also be accessed by administrators in the web interface under _Adminsiter -> Settings_
+* HTTP(S) proxy - hostname, port and optional credentials in format of `http://[user:password@]HOST:PORT`
+* HTTP(S) proxy except hosts - array of hostnames to exclude
```
Comment (evgeni) [inline _includes/manuals/nightly/5.9_http_proxy.md]: how do I pass an array? `['host1.example.com', 'host2.example.com']`? Or comma separated?

**Comment by lzap** [comment]
All done.

**Comment by mmoll** [comment]
merged, díky @lzap!

**Pull request merged**